README.md

```diff
@@ -96,6 +96,7 @@ See [server demo](example) and [browser demo](https://github.com/bcherny/json-sc
 | unknownAny | boolean | `true` | Use `unknown` instead of `any` where possible |
 | unreachableDefinitions | boolean | `false` | Generates code for `$defs` that aren't referenced by the schema. |
 | $refOptions | object | `{}` | [$RefParser](https://github.com/BigstickCarpet/json-schema-ref-parser) Options, used when resolving `$ref`s |
+| expandPropertyTitles | boolean | `true` | Use the `"title"` field within each property to expand into a separate type? |
```

Comment:
I see so for me it's not clear to in terms of type system what exactly we going to expand so I would propose to use this naming which is closer to the TS type system:  `ignoreLiteralTypeInTitle` or `useStringTypeInTitle`

Reply:
How do you feel about `useTitleAsType`? It's not necessarily a "literal type" versus a "string type"?

Reply:
ok last second idea - maybe just `useStringTypeForTitle` should be very clear